src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -127,18 +131,28 @@ export const ContentMessageComponent = ({
 
   const isConversationReadonly = conversation.readOnlyState() !== null;
 
+  const contentMessageWrapperRef = (element: HTMLDivElement | null) => {
+    setTimeout(() => {
+      if (element?.parentElement?.querySelector(':hover') === element) {
+        // Trigger the action menu in case the component is rendered with the mouse already hovering over it
+        setActionMenuVisibility(true);
+      }
+    });
+  };
+
+  const asset = assets?.[0] as FileAssetType;
```

Comment:
question: Shouldn't we add `| undefined` there? We're not sure it exists at this point...